### DISCUSSION TITLE
Allow CONNECT HTTP method on Ingress

### DISCUSSION THREAD

**Discussion by damdinsharav** [rancher/rke2#5485] [Q&A]
Hi, I have application that uses CONNECT method. Ingress currently returns 405. Is there way to allow CONNECT method?

`[17/Feb/2024:00:51:35 +0000] "CONNECT /application/api/peer HTTP/1.1" 405 150 "-" "-" 375 0.000 [] [] - - - - --7061c11f8595f17daaf142f68720e330`

**Comment by brandond** [comment]
Rke2 ships with ingress-nginx. I would check out the ingress-nginx docs:
https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/
